### PR TITLE
feat: Diffable formater: Attribute Per Line functionality

### PR DESCRIPTION
**Pull request by mauryapari**
Related to issue #10

**Notes for reviewer:**

* Contains code to implement attributePerLine functionality based on formatting object passed.
* Contains test cases for the smae

**Checklist:**

* [ ] Update dependencies
* [ ] Bump

### CODE REVIEW

File: src/loadOptions.js [hunk 1]
```diff
@@ -97,6 +97,12 @@ export const loadOptions = function () {
     if (typeof(globalThis.vueSnapshots.formatting.selfClosingTag) !== 'boolean') {
       globalThis.vueSnapshots.formatting.selfClosingTag = false;
     }
+    if (
+      typeof(globalThis.vueSnapshots.formatting.attributesPerLine) !== 'number' || 
+      globalThis.vueSnapshots.formatting.attributesPerLine < 0
+    ) {
+      globalThis.vueSnapshots.formatting.attributesPerLine = 1;
+    }
```
[the remark this answers is not in the document]
Comment (mauryapari) [reply src/loadOptions.js:106]: Added a check for whole numbers

File: src/formatMarkup.js [hunk 2]
```diff
@@ -147,33 +150,30 @@ export const diffableFormatter = function (markup, options) {
...
-      result = result + '\n' + '  '.repeat(indent) + endingAngleBracket.trim();
+        if (isNewLine) {
+          return '\n' + '  '.repeat(indent + 1) + attrVal;
+        } else {
+          return ' ' + attrVal;
+        }
+      }).join('');
+  
+      if (node.attrs.length <= options.attributesPerLine) {
+        result += `${formattedAttr}${endingAngleBracket}`;
```
Comment (TheJaredWilcurt) [inline src/formatMarkup.js:173]: less syntactical noise.
```suggestion
        result += formattedAttr + endingAngleBracket;
```

File: tests/unit/src/loadOptions.test.js [hunk 3]
```diff
@@ -204,4 +205,28 @@ describe('Load options', () => {
...
+      [true, 1],
+      [100, 100],
+      [7.5, 1],
+      [null, 1]
+    ];
+
+    test.each(testCases)('Attributes per line when value is "%s"', (value, expected) => {
+      globalThis.vueSnapshots.formatting.attributesPerLine = value;
+      loadOptions();
+      expect(global.vueSnapshots.formatting.attributesPerLine)
```
Comment (TheJaredWilcurt) [inline tests/unit/src/loadOptions.test.js:228]:
```suggestion

      expect(global.vueSnapshots.formatting.attributesPerLine)
```

File: src/loadOptions.js [hunk 4]
```diff
@@ -97,6 +97,13 @@ export const loadOptions = function () {
     if (typeof(globalThis.vueSnapshots.formatting.selfClosingTag) !== 'boolean') {
       globalThis.vueSnapshots.formatting.selfClosingTag = false;
     }
+    if (
+      typeof(globalThis.vueSnapshots.formatting.attributesPerLine) !== 'number' || 
+      globalThis.vueSnapshots.formatting.attributesPerLine < 0 ||
+      globalThis.vueSnapshots.formatting.attributesPerLine % 1 !== 0
+    ) {
```
Comment (TheJaredWilcurt) [inline src/loadOptions.js:104]:
```suggestion
    ) {
      if (globalThis.vueSnapshots.formatting.attributesPerLine !== undefined) {
        logger([
          'global.vueSnapshots.formatting.attributesPerLine',
          'must be a whole number.'
        ].join(' '));
      }
```

**Pull request merged**